src/Components/CreateImageWizard/ValidatedTextInput.tsx

```diff
@@ -30,6 +34,45 @@ interface HookValidatedTextInputPropTypes extends TextInputProps {
   warning?: string;
 }
 
+export const HookPasswordValidatedInput = ({
+  ariaLabel,
+  value,
+  stepValidation,
+  fieldName,
+  ...rest
```

Comment:
Generally in the code base we try to avoid prop spreading. It makes things less explicit. The official React docs even caution to "use spread syntax with restraint": https://react.dev/learn/passing-props-to-a-component#forwarding-props-with-the-jsx-spread-syntax

Would you mind eliminating the prop spreading here?